packages/bpk-component-table/src/BpkTableCell-test.js

```diff
@@ -22,7 +22,7 @@ import BpkTableCell from './BpkTableCell';
 
 describe('BpkTableCell', () => {
   it('should render correctly', () => {
-    const tree = renderer.create(<BpkTableCell>Tabular data</BpkTableCell>).toJSON();
+    const tree = renderer.create(<BpkTableCell className="test">Tabular data</BpkTableCell>).toJSON();
```

Comment:
Hi, that should be the additional test added. I've also added the changelog entry as well.